tests/specs/open-db.js

```diff
@@ -56,6 +56,49 @@
             });
         });
 
+        it('should normally reject open promise with store conflicting with Server methods', function (done) {
+            db.open({
+                server: dbName,
+                version: 1,
+                schema: {
+                    query: {
+                        key: {
+                            keyPath: 'id'
+                        }
+                    }
+                }
+            }).catch(function (err) {
+                expect(err.toString()).toContain('conflicts with db.js method');
+                done();
+            });
+        });
+
+        it('should not add stores to server using noServerMethods', function (done) {
+            var spec = this;
+            db.open({
+                server: dbName,
+                version: 1,
+                noServerMethods: true,
+                schema: {
+                    test: {
+                        key: {
+                            keyPath: 'id'
+                        }
+                    },
+                    query: {
+                        key: {
+                            keyPath: 'id'
+                        }
+                    }
+                }
+            }).then(function (s) {
+                spec.server = s;

```

Comment:
I set spec.server so that the `afterEach` had the server to close...Otherwise, the next test got blocked...